amazon_sel.py

```diff
@@ -4,14 +4,14 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.common.by import By
 
-class Amazon :
-    def __init__(self) :
-        self.item = None
-        self.base_url = 'https://www.amazon.in' 
-        self.browser = webdriver.Chrome(executable_path='./../drivers/chromedriver.exe')
 
-    def search_text(self, item) :
+class Amazon:
+    def __init__(self):
+        self.item = None
+        self.base_url = 'https://www.amazon.in'
+        self.browser = webdriver.Chrome()

```

Comment:
Change here: I pulled this on my linux machine and chromedriver.exe doesn't exist. I think it would be easier to assume people have set up the correct pathing (a normal install worked for me) instead of specifying a location that might not be correct.

Your path also assumes a specific relative code folder location.